applications/browser/package.json

```diff
@@ -85,37 +85,18 @@
     "@theia/vsx-registry": "1.39.0",
     "@theia/workspace": "1.39.0",
     "fs-extra": "^9.0.1",
-    "theia-blueprint-product": "1.39.0"
+    "blueprint-product-ext": "1.39.0"
   },
   "devDependencies": {
     "@theia/cli": "1.39.0"
   },
   "scripts": {
-    "prepare": "yarn build",
     "clean": "theia clean && rimraf node_modules",
-    "build": "yarn -s compile && yarn -s bundle",
-    "bundle": "yarn -s rebuild && theia build",
-    "compile": "tsc -b",
-    "rebuild": "theia rebuild:browser --cacheRoot ../..",
-    "start": "theia start --plugins=local-dir:./plugins",
-    "watch": "concurrently --kill-others -n tsc,bundle -c red,yellow \"tsc -b -w --preserveWatchOutput\" \"yarn -s watch:bundle\"",
-    "download:plugins": "theia download:plugins --rate-limit=15",
+    "build": "yarn -s rebuild && theia build --app-target=\"browser\" --mode development",
+    "build:prod": "theia  build --app-target=\"browser\"",
```

Comment:
Do we need a rebuild here as well? I though this has to be done before `theia build` since 1.39.0

Reply:
added, thanks!